modules/shutter_page/custom_domains.tf

```diff
@@ -0,0 +1,18 @@
+# data "template_file" "customdomain" {
+#   template = "${file("${path.cwd}/templates/custom_domain.json")}"
+# }
+
+# resource "azurerm_template_deployment" "custom_domain" {
```

Comment:
this isn't working?

Reply:
It was working earlier but last night I saw some errors 

> "Sorry, it looks like there was an error on our end. Please contact Support if you keep having this problem."

So I have commented this bit and will enable this but once I have a reply from MS.